js/main.js

```diff
@@ -23,7 +24,29 @@ $displayEntriesLi.addEventListener('click', handleLiClick);
 $aDelete.addEventListener('click', deleteClick);
 $cancelBtn.addEventListener('click', cancelBtnClick);
 $deleteBtn.addEventListener('click', deleteConfirmClick);
+$searchTitle.addEventListener('input', searchTitleInput);
 
+function searchTitleInput(event) {
+  if (event.target.value.length === 0) {
+    for (var l = 0; l < $liElements.length; l++) {
+      $liElements[l].setAttribute('class', 'show');
+    }
+    return;
+  }
+  var arr = [];
+  for (var j = 0; j < data.entries.length; j++) {
+    if (data.entries[j].title.toLowerCase().indexOf(event.target.value) !== -1) {
+      arr.push(data.entries[j].entryId);
+    }
+  }
+  for (var i = 0; i < $liElements.length; i++) {
+    if (arr.indexOf(parseInt($liElements[i].getAttribute('data-entry-id'))) !== -1) {
+      $liElements[i].setAttribute('class', 'show');
+    } else {
+      $liElements[i].setAttribute('class', 'hidden');
+    }
+  }
```

Comment:
I would try to do this with one loop by looping over the list of li elements and checking if either the title or notes contain the searched phrase. If neither the title or notes include the searched phrase, you should hide it.